src/utils/emoji.js

```diff
@@ -10,10 +10,11 @@ const regex = emojiRegex();
 
 const reflect = promise => (...args) => promise(...args).then(v => v, e => e);
 
-const fetchEmojiImage = reflect(resolveImage);
+const makeFetchEmojiImage = () => reflect(resolveImage);
 
 const getCodePoints = string =>
   Array.from(string)
+    .filter(x => x !== '️')
```

Comment:
And this 😅

Reply:
Yes, this is not really obvious, I will add a comment. This is the fix. The picture below should make it clearer.

![image](https://user-images.githubusercontent.com/13083475/52967688-d12d2680-3378-11e9-9315-1403538e2536.png)